diff --git a/node_modules/typescript/lib/tsc.js b/node_modules/typescript/lib/tsc.js
index 721fde8..1c85849 100644
--- a/node_modules/typescript/lib/tsc.js
+++ b/node_modules/typescript/lib/tsc.js
@@ -5,16 +5,6 @@ import { execFileSync } from "node:child_process";
 
 const exe = getExePath();
 
-if (process.platform !== "win32" && typeof process.execve === "function") {
-    // > v22.15.0
-    try {
-        process.execve(exe, [exe, ...process.argv.slice(2)]);
-    }
-    catch {
-        // may not be available, ignore the error and fallback
-    }
-}
-
 try {
     execFileSync(exe, process.argv.slice(2), { stdio: "inherit" });
 }
